### PR TITLE
Add Angular logo

### PR DESCRIPTION
**Pull request by Nabagata**
- [ ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [ ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/russian/angularjs/index.md [hunk 1]
```diff
@@ -26,5 +27,3 @@ HTML:
 ```
 
 HTML
-
-\`\` \`
```
Comment (1andee) [inline guide/russian/angularjs/index.md]: @cmccormack I think this is supposed to somehow tie to line 14? It isn't markdown syntax that I have seen before.

**Comment by RandellDawson** [comment]
This PR was closed and then re-opened in an attempt to clear a specific Travis build failure.

File: guide/russian/angularjs/index.md (hunk 1 above)
Comment (cmccormack) [reply guide/russian/angularjs/index.md]: @1andee you are correct, I don't know what the use of either line is but I reverted my commit.

**Comment by cmccormack** [comment]
This looks good to me.  Thank you for your contribution!

**Pull request merged**